tools/windows_buildenv.bat

```diff
@@ -172,9 +171,12 @@ REM Generate CMakeSettings.json which is read by MS Visual Studio to determine t
     >>%CMakeSettings% echo       "generator": "Ninja",
     >>%CMakeSettings% echo       "inheritEnvironments": [ "msvc_!PLATFORM!_!PLATFORM!" ],
     >>%CMakeSettings% echo       "installRoot": "!INSTALL_ROOT:\=\\!\\${name}",
+    >>%CMakeSettings% echo       "cmakeToolchain": "!BUILDENV_PATH!\scripts\buildsystems\vcpkg.cmake",
```

Comment:
>>%CMakeSettings% echo       "cmakeToolchain": "!BUILDENV_PATH:\=\\!\\scripts\\buildsystems\\vcpkg.cmake",

Reply:
Use double \\ Escap character and backslash itself

Reply:
fixed

Reply:
fixed and force pushed

Reply:
This change got lost

Reply:
Whoops. Fixed again and force pushed. Please test again.